server/server.js

```diff
@@ -1,81 +1,98 @@
-//need to create actual routes that will 1. receive a fetch request from the frontend 
-//2. send information to the database to save 
+//need to create actual routes that will 1. receive a fetch request from the frontend
+//2. send information to the database to save
 //3. parse through the information (from the db query) and send that information back to the frontend so we can use it to change the state
 
-const express = require('express');
-const path = require('path');
-const cloudinary = require('cloudinary')
-const formData = require('express-form-data')
+const mongoose = require("mongoose");
+const express = require("express");
+const path = require("path");
+const cloudinary = require("cloudinary");
+const formData = require("express-form-data");
 
 const PORT = 3000;
 
 const app = express();
 
-const userController = require('./controllers/controller');
+const userController = require("./controllers/controller");
```

Comment:
please name the file userController if you are going to modify only User related transactions.

Reply:
we haven't touched that file yet :)